services/QuillLMS/spec/models/question_spec.rb

```diff
@@ -526,4 +526,105 @@
     end
   end
 
+  describe 'create_translation_mappings' do
+    subject {question.create_translation_mappings}
+
+    let(:question) { create(:question, data: {'instructions' => instructions})}
+    let(:instructions) { "do this question"}
+    let(:english_texts ) { question.english_texts }
+
+    it 'creates translation_mappings for focusPoints and incorrectSequences' do
+      expect(question).to receive(:create_data_translation_mappings).with(type: Question::FOCUS_POINTS)
+      expect(question).to receive(:create_data_translation_mappings).with(type: Question::INCORRECT_SEQUENCES)
+      subject
+    end
+
+    it 'creates translation_mappings for instructions' do
+      subject
+      instructions_mapping = question.translation_mappings.find_by(field_name: "instructions")
+      expect(instructions_mapping.text).to eq(instructions)
+    end
```

Comment:
This may not actually need a change, but I did want to point out that it feels a little weird to test for mappings in two different ways.  For FocusPoints and IncorrectSequences we confirm that stubbed methods are called, but we check for database artifacts to be created for Instructions.  Given that you have coverage for `create_data_translation_mappings` below, that may be just fine, but I wanted to call out that this tweaked a sort of "that seems weird" moment from me.